PlayerMatcher/Matchmaker/MatchmakerDocumentation.md

```diff
@@ -0,0 +1,11 @@
+The baseline matchmaking algorithm is based on the Elo rating system for determining player selection for a match. At the start the matchmaker 
```

Comment:
if you rename this file readme.md it will automatically display when you open the fiolder in github.